html/ui/classic/js/brs.modals.account.js

```diff
@@ -273,6 +273,12 @@ var BRS = (function(BRS, $, undefined) {
 			case 1:
 			    transactionType = "AT Payment";
 			    break;
+			case 2:
+				transactionType = "AT Asset Mold";
```

Comment:
What is the "Mold" thing? Is it the "Issuance" or "AT Asset Issue"?

Reply:
Yes, it is AT asset issuance.